backend/Dockerfile.scheduledworkflow

```diff
@@ -30,15 +30,19 @@ RUN go-licenses csv ./backend/src/crd/controller/scheduledworkflow > /tmp/licens
     diff /tmp/licenses.csv backend/third_party_licenses/swf.csv && \
     go-licenses save ./backend/src/crd/controller/scheduledworkflow --save_path /tmp/NOTICES
 
-FROM alpine:3.8
+FROM alpine:3
```

Comment:
Same as in [here](https://github.com/kubeflow/pipelines/pull/8944/files#r1131848819)

Reply:
same as in [here](https://github.com/kubeflow/pipelines/pull/8944#discussion_r1131862439)